woodwork/column_accessor.py

```diff
@@ -196,10 +196,13 @@ def wrapper(*args, **kwargs):
 
                 # Try to initialize Woodwork with the existing schema
                 if _is_series(result):
-                    valid_dtype = _get_valid_dtype(type(result), self._schema['logical_type'])
+                    valid_dtype = _get_valid_dtype(type(result), self._schema.logical_type)
                     if str(result.dtype) == valid_dtype:
-                        schema = copy.deepcopy(self._schema)
-                        result.ww.init(**schema, use_standard_tags=self.use_standard_tags)
+                        result.ww.init(logical_type=self._schema.logical_type,
```

Comment:
This is the first place where we could benefit from being able to do `new_series.ww.init(schema=series.ww.schema, use_standard_tags=series.ww.use_standard_tags)` like how we allow it in the table accessor. 

Since there's only four init params that we pass in, I might say we can get by without it for simplicity's sake, but we do this call four times by my count, and we also need to remember to copy the elements each time, and I think we'll be less likely to add a bug in the future if we are able to simplify this type of call.